blog/2020-12-09-eclipse-vert-x-4-released.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Eclipse Vert.x 4 released!
+category: releases
+authors:
+  - name: Julien Viet
+    github_id: vietj
+summary: >-
+  Vert.x 4 has just been released and it's the best thing since sliced buffers.
+---
+
+We are extremely pleased to announce the release of Vert.x 4.
+
+## What's new in Vert.x 4?
+
+Vert.x 4 is the evolution of the Vert.x 3.x series that brings [key features](https://vertx.io/blog/whats-new-in-vert-x-4/) to Vert.x.
+
+## From Vert.x 3 to Vert.x 4
+
+Vert.x 4 extends the Vert.x 3 line with a set of new features while providing the same Vert.x 3 experience:
+Vert.x 3 users will feel at home with Vert.x 4.
+
+Vert.x 3 users migrating to Vert.x 4 should read this [page](https://vertx.io/blog/from-vert-x-3-to-vert-x-4/).
```

Comment:
not page but "this migration guide"

Reply:
actually it is a page that talks about migration which contains the guide but not only, there is a broader message sent in this page

Reply:
we can simply say "You can learn more about migration [here](https://vertx.io/blog/from-vert-x-3-to-vert-x-4/)."